src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+const galleryList = document.querySelector('.gallery__list');
+const mainPic = document.getElementById('largeImg');
+
+galleryList.addEventListener('click', (e) => {
+  e.preventDefault();
+  mainPic.src = e.target.src;
```

Comment:
Consider adding a check to ensure that `e.target` is an image element before setting `mainPic.src`. This will prevent errors if the user clicks on a non-image element within `galleryList`.